en/sensor_bus/i2c.md

```diff
@@ -1 +1,38 @@
-# I2C Bus Overview
\ No newline at end of file
+# I2C Bus Overview
```

Comment:
@bkueng This section updated. Is it OK now?

Reply:
👍